lib/webpacker/manifest.rb

```diff
@@ -28,24 +28,24 @@ def lookup!(name)
   end
 
   private
-    def compiling?
-      config.compile? && !dev_server.running?
-    end
+  def compiling?
```

Comment:
Please can you revert indentation change. All rails gems use this style: 

```rb
private
  def compiling?
  end

Reply:
May be just run `bundle exec rubocop -a` to fix

Reply:
OK - will do

Reply:
RubyMine get whiny about double quotes when singles will do... Interesting